src/components/__tests__/ListItem.spec.tsx

```diff
@@ -66,9 +79,30 @@ describe('ListItem', () => {
       expect(getByText(title).instance.props.style[0].color).toBe(COLORS.text)
     })
   })
+  it('should handle long press with locked exercise', async () => {
```

Comment:
Please remove this test, it does not test anything at all, which is not tested by other tests in this file.